src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -30,31 +28,31 @@ public SwerveDriveMap getSwerveDriveMap() {
         // All Distances are in Meters
         // Front Left Module
         final CANCoder encoderFL = new CANCoder(1);
-        encoderFL.configMagnetOffset(0); // TODO Get Magnet Offset
+        encoderFL.configMagnetOffset(-195.381);
```

Comment:
Please look [here](https://www.chiefdelphi.com/t/official-sds-mk3-mk4-code/397109/96) for some info that might help with this.

Reply:
What do you mean by this?

Reply:
Seems to not be as useful as I was expecting - this is fine